openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -78,7 +78,7 @@ def get_value_for_org(cls, org, name, default=None):
         """
         for configuration in cls.objects.filter(values__contains=org, enabled=True).all():
             org_filter = configuration.get_value('course_org_filter', None)
```

Comment:
should this be a call to the new method instead of the direct setting reference?